rosdep/base.yaml

```diff
@@ -1119,7 +1119,10 @@ golang-go:
   gentoo: [dev-lang/go]
   ubuntu: [golang-go]
 google-mock:
-  alpine: [gmock-dev]
+  alpine:
+    '*': [gtest-dev, gmock]
```

Comment:
In official repository, dev contents of gmock is in gtest-dev. (available since Alpine v3.9)